src/www/cordova_index.html

```diff
@@ -22,19 +22,11 @@
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=no" />
     <link rel="stylesheet" href="style.css" />
     <!-- See https://github.com/webcomponents/custom-elements#es5-vs-es2015  -->
-    <script src="bower_components/webcomponentsjs/custom-elements-es5-adapter.js "></script>
+    <script src="node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
```

Comment:
We are not transpiling to ES5. Remove.

Reply:
We do transpile to ES5 - that's the default babel env preset.

Reply:
We should change to ES2015 then. It may solve some issues. We're using ES2015 in the manager.

Reply:
I left the Babel transpiling step to remain consistent with the current build.

We can remove that and target ES2015 if we want to drop support for older Android versions.